implementation/src/main/java/com/teampc/controller/SubmissionRowController.java

```diff
@@ -61,17 +62,13 @@ void onViewDetailsClickHandler(ActionEvent event) {
       Test currentSelection = currentTest;
 
       try {
-         FXUtils.switchToScreenAndConfigureController((Stage) ((Node) event.getSource()).getScene().getWindow(), "take-test.fxml",
+         FXUtils.switchToScreenAndConfigureController((Stage) ((Node) event.getSource()).getScene().getWindow(), "submission-results.fxml",
             (controller, stage) -> {
```

Comment:
just fyi, if you change the params to `(SubmissionResultController controller, Stage stage) -> {` then you don't need to cast later